src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -339,15 +339,38 @@ void CC32xxWiFi::wlan_mac(uint8_t mac[6])
     sl_NetCfgGet(SL_MAC_ADDRESS_GET, nullptr, &len, mac);
 }
 
+void *vSimpleLinkSpawnTask(void *pvParameters)
```

Comment:
What's the difference between this implementation and VStartSimplelinkSpawnTask ?